content/en/docs/started/installing-kubeflow.md

```diff
@@ -158,6 +158,17 @@ The following table lists <b>active distributions</b> that have <b>had a recent
           1.6.1
         </td>
       </tr>
+      <tr>
+        <td>Kubeflow on QBO</td>
+        <td>QBO</td>
+        <td>QBO Kubernetes Engine Community Edition (QKE CE), QBO Kubernetes Engine Cloud (QKE Cloud)</td>
+        <td>
+          <a href="https://docs.qbo.io/#/ai_and_ml?id=kubeflow">Website</a>
```

Comment:
Is this the best link for people to learn about your tool?

Reply:
Yes that is the best link to learn about Qbo and Kubeflow